services/121-service/src/programs/dto/program-return.dto.ts

```diff
@@ -94,9 +94,9 @@ export class ProgramReturnDto {
   @ApiProperty({ example: 1, type: 'number' })
   id: number;
 
-  @ApiProperty({ example: false })
-  @IsBoolean()
-  public readonly published: boolean;
+  // @ApiProperty({ example: false })
+  // @IsBoolean()
+  // public readonly published: boolean;
```

Comment:
You are heading in the right direction, this and your other commented out code can go 👍